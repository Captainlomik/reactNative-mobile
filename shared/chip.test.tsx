import * as renderer from "react-test-renderer";
import { Chip } from "./chip";

describe('<Chip />', () => {
  it("simple chip", () => {
    const tree = renderer.
      create(<Chip text="hello ma old friend" />).
      toJSON()

    expect(tree).toMatchSnapshot()
  })
})
