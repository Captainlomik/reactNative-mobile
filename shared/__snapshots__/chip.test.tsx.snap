// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chip /> simple chip 1`] = `
<View
  style={
    {
      "borderColor": "#4D5064",
      "borderRadius": 17,
      "borderWidth": 1,
      "paddingHorizontal": 10,
      "paddingVertical": 5,
    }
  }
>
  <Text
    style={
      {
        "color": "#FAFAFA",
        "fontFamily": "FiraSans",
        "fontSize": 14,
      }
    }
  >
    hello ma old friend
  </Text>
</View>
`;
